packages/library/src/send-transaction.ts

```diff
@@ -55,6 +60,23 @@ type SendAndConfirmDurableNonceTransactionFunction = (
     >,
 ) => Promise<void>;
 
+type SendTransactionWithoutConfirmingFunction = (
+    transaction: SendableTransaction,
+    config: Omit<Parameters<typeof sendTransaction_INTERNAL_ONLY_DO_NOT_EXPORT>[0], 'rpc' | 'transaction'>,
+) => Promise<void>;
+
+export function sendTransactionWithoutConfirmingFactory({
+    rpc,
+}: SendTransactionWithoutConfirmingFactoryConfig): SendTransactionWithoutConfirmingFunction {
+    return async function sendTransactionWithoutConfirming(transaction, config) {
+        await sendTransaction_INTERNAL_ONLY_DO_NOT_EXPORT({
```

Comment:
I'm seeing how long I can go without returning the signature from any of the `send*` methods. I want people to start to get the signature _ahead_ of time using `getSignatureFromTransaction()` so that they can start to make use of it in their apps sooner.

Reply:
The harbinger of JavaScript-Solana pain.